### DISCUSSION TITLE
Manager: remote include key support for gitlabci-include

### DISCUSSION THREAD

**Discussion by soullivaneuh** [renovatebot/renovate#25880] [Suggest an Idea]
### Tell us more.

According to the current state of the [gitlabci-include manager' spec](https://github.com/renovatebot/renovate/blob/46d1569f76ca3114f9766aced8d1a0240a71892b/lib/modules/manager/gitlabci-include/extract.spec.ts#L50-L53), the following include method using `remote` is simply ignored:

```yaml
include:
  - remote: https://gitlab.com/ci-dedicated-project/-/raw/v1.2.3/the-file.gitlab-ci.yml
```

This what I had to explain on my personal GitLab CI recipes library: https://gitlab.com/ss-open/ci/recipes/-/blob/f36e8ad9d15847459552bb47ba78c31d3eabf288/README.md#include-methods-for-projects-outside-of-gitlabcom

However, this kind of include is may be necessary when you need to include a gitlab.com CI library from a self-hosted GitLab instance.

So it would be a great addition to support the `remote` include for that.

What do you think? Do you have any limitations to achieve that?

Thanks!

**Comment by viceice** [comment]
Should be done in `gitlabci` manager instead of `gitlabci-include`, the latter will be merged back to the former.

PR welcome
